bundles/org.eclipse.rap.rwt/src/org/eclipse/swt/events/ControlListener.java

```diff
@@ -41,4 +43,36 @@ public interface ControlListener extends SWTEventListener {
    * @param e an event containing information about the resize
    */
   void controlResized( ControlEvent e );
+  
+  /**
+   * Static helper method to create a <code>ControlListener</code> for the
+   * {@link #controlMoved(ControlEvent e)}) method, given a lambda expression or a method reference.
+   *
+   * @param c the consumer of the event
+   * @return ControlListener
```

Comment:
As this is a new API in RAP, please add @since 4.1 tag in the JavaDoc everywhere.